repo.go

```diff
@@ -830,6 +825,12 @@ func (r *Repo) Snapshot() error {
 	return r.SnapshotWithExpires(data.DefaultExpires("snapshot"))
 }
 
+func (r *Repo) snapshotManifests() []string {
```

Comment:
We moved away from using the term `manifests` to `metadata` in https://github.com/theupdateframework/go-tuf/pull/154 to better align with specification terminology.